exercises/atbash-cipher/example.rs

```diff
@@ -0,0 +1,39 @@
+fn ascii(ch: char) -> u8 {
+    ch as u8
+}
+
+fn get_transpose(ch: char) -> char {
+    if ch.is_digit(10) {
+        ch
+    } else {
+        (ascii('z') - ascii(ch) + ascii('a')) as char
+    }
+}
+
+pub fn encode(plaintext: &str) -> String {
+    plaintext
+    .to_lowercase()
+    .chars()
+    .filter(|&ch| ch.is_alphanumeric())
+    .filter(|&ch| (ch as u8) < 128)
+    .map(|ch| get_transpose(ch))
+    .fold("".to_string(),|encoded_result, ch| {
```

Comment:
This works as a way to split the encoded string into groups of 5. Another way would be to build the string without spaces and then use `chunks` to split into 5-character chunks.

https://doc.rust-lang.org/std/primitive.slice.html#method.chunks

You may also be able to do the same thing with `split`